### ISSUE TITLE
Kernel files and snapshots contain absolute paths from developer's machine

### ISSUE THREAD

**Issue by bkonyi** [flutter/flutter#22593]
Tracking issue for dart-lang/sdk#34654

It appears that kernel files are capturing paths from the machine they were generated on. For example, I found this path in the kernel file and AOT compilation output of the android_alarm_manager example application: file:///usr/local/google/home/bkonyi/plugins/packages/android_alarm_manager/example/lib/main.dart

At the very least, fully qualified paths like this could add a number of KB to a shipped Flutter application and could be considered a privacy concern due to personally identifying information potentially being part of the path.

cc/ @Hixie @matthew-carroll

**Comment by pedromassangocode** [comment]
Is this still a valid issue today?
This and the issue mentioned are still opened but without any recent interactions.

Maybe this need a proper priority label!

**Comment by bkonyi** [comment]
Yes this is still a valid issue. Probably not high priority if it's been open for this long with no action.
